contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -55,7 +55,7 @@ cd .. && \
 #
 # bpftool
 #
-git clone --depth 1 -b bpftool https://github.com/cilium/linux.git && \
+git clone --depth 1 -b master git://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git \
```

Comment:
It failed to build this image on quay:

```
2/27/2020, 10:06:34 AMsed "s|@SYSCONFDIR@|/etc/iproute2|g" ip-route.8.in > ip-route.8
2/27/2020, 10:06:34 AMfatal: Too many arguments. usage: git clone [<options>] [--] <repo> [<dir>]
2/27/2020, 10:06:34 AM
2/27/2020, 10:06:34 AM
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-v
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--verbose
2/27/2020, 10:06:35 AMbe more verbose
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-q
2/27/2020, 10:06:35 AM, --quiet
2/27/2020, 10:06:35 AMbe more quiet
2/27/2020, 10:06:35 AM--progress
2/27/2020, 10:06:35 AMforce progress reporting
2/27/2020, 10:06:35 AM-n
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--no-checkout
2/27/2020, 10:06:35 AMdon't create a checkout
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--bare
2/27/2020, 10:06:35 AMcreate a bare repository
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--mirror
2/27/2020, 10:06:35 AMcreate a mirror repository (implies bare)
2/27/2020, 10:06:35 AM-l
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--local
2/27/2020, 10:06:35 AMto clone from a local repository
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--no-hardlinks
2/27/2020, 10:06:35 AMdon't use local hardlinks, always copy
2/27/2020, 10:06:35 AM-s
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--shared
2/27/2020, 10:06:35 AMsetup as shared repository
2/27/2020, 10:06:35 AM--recurse-submodules
2/27/2020, 10:06:35 AM[=<pathspec>]
2/27/2020, 10:06:35 AMinitialize submodules in the clone
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-j
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--jobs
2/27/2020, 10:06:35 AM<n>
2/27/2020, 10:06:35 AMnumber of submodules cloned in parallel
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--template
2/27/2020, 10:06:35 AM<template-directory>
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AMdirectory from which templates will be used
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--reference
2/27/2020, 10:06:35 AM<repo>
2/27/2020, 10:06:35 AMreference repository
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--reference-if-able
2/27/2020, 10:06:35 AM<repo>
2/27/2020, 10:06:35 AMreference repository
2/27/2020, 10:06:35 AM--dissociate
2/27/2020, 10:06:35 AMuse --reference only while cloning
2/27/2020, 10:06:35 AM-o
2/27/2020, 10:06:35 AM, --origin
2/27/2020, 10:06:35 AM<name>
2/27/2020, 10:06:35 AMuse <name> instead of 'origin' to track upstream
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-b,
2/27/2020, 10:06:35 AM--branch
2/27/2020, 10:06:35 AM<branch>
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AMcheckout <branch> instead of the remote's HEAD
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-u
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--upload-pack
2/27/2020, 10:06:35 AM<path>
2/27/2020, 10:06:35 AMpath to git-upload-pack on the remote
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--depth
2/27/2020, 10:06:35 AM<depth>
2/27/2020, 10:06:35 AMcreate a shallow clone of that depth
2/27/2020, 10:06:35 AM--shallow-since
2/27/2020, 10:06:35 AM<time>
2/27/2020, 10:06:35 AMcreate a shallow clone since a specific time
2/27/2020, 10:06:35 AM--shallow-exclude
2/27/2020, 10:06:35 AM<revision>
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AMdeepen history of shallow clone, excluding rev
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--single-branch clone only one branch, HEAD or --branch
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--no-tags
2/27/2020, 10:06:35 AMdon't clone any tags, and make later fetches not to follow them
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM--shallow-submodules
2/27/2020, 10:06:35 AMany cloned submodules will be shallow
2/27/2020, 10:06:35 AM--separate-git-dir <gitdir>
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AMseparate git dir from working tree
2/27/2020, 10:06:35 AM-c
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--config
2/27/2020, 10:06:35 AM<key=value>
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AMset config inside the new repository
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-4
2/27/2020, 10:06:35 AM,
2/27/2020, 10:06:35 AM--ipv4
2/27/2020, 10:06:35 AMuse IPv4 addresses only
2/27/2020, 10:06:35 AM
2/27/2020, 10:06:35 AM-6
2/27/2020, 10:06:35 AM, --ipv6
2/27/2020, 10:06:35 AMuse IPv6 addresses only
2/27/2020, 10:06:35 AM--filter
2/27/2020, 10:06:35 AM<args>
2/27/2020, 10:06:35 AMobject filtering
2/27/2020, 10:06:35 AM
```

It's mostly likely related with the removal of ` && \`